ReorderableList conflict in 2020.3.22f1+

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->
On Unity 2020.3.22f1 (and assuming later versions as well)

On install:
Library\PackageCache\com.codewriter.triinspector@692dfb6620\Unity.InternalAPIEditorBridge.012\ReorderableListProxy.cs(42,18): error CS1061: 'ReorderableList' does not contain a definition for 'InvalidateCacheRecursive' and no accessible extension method 'InvalidateCacheRecursive' accepting a first argument of type 'ReorderableList' could be found (are you missing a using directive or an assembly reference?)

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->
No errors should present.

**Code Sample**
<!-- If applicable, add code sample to help explain your problem. -->
```csharp
To fix, in ReorderableListProxy:41, remove the bit that says UNITY_2020_3
```

--- a/Unity.InternalAPIEditorBridge.012/ReorderableListProxy.cs
+++ b/Unity.InternalAPIEditorBridge.012/ReorderableListProxy.cs
@@ -1,3 +1,5 @@
+using System;
+using System.Reflection;
 using UnityEditorInternal;
 using UnityEngine;
 
@@ -5,6 +7,10 @@ namespace TriInspectorUnityInternalBridge
 {
     internal static class ReorderableListProxy
     {
+#if !UNITY_2021_3_OR_NEWER
+        private static readonly MethodInfo ClearCacheMethod;
+#endif
+
         private static ReorderableList.Defaults _defaultBehaviours;
 
         // ReSharper disable once InconsistentNaming
@@ -19,6 +25,15 @@ namespace TriInspectorUnityInternalBridge
 
                 return _defaultBehaviours;
             }
+        }
+
+        static ReorderableListProxy()
+        {
+#if !UNITY_2021_3_OR_NEWER
+            const BindingFlags flags = BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance;
+            ClearCacheMethod = typeof(ReorderableList).GetMethod("InvalidateCacheRecursive", flags) ??
+                               typeof(ReorderableList).GetMethod("ClearCacheRecursive", flags);
+#endif
         }
 
         public static void DoListHeader(ReorderableList list, Rect headerRect)
@@ -38,10 +53,10 @@ namespace TriInspectorUnityInternalBridge
 
         public static void ClearCacheRecursive(ReorderableList list)
         {
-#if UNITY_2021_3_OR_NEWER || UNITY_2020_3
+#if UNITY_2021_3_OR_NEWER
             list.InvalidateCacheRecursive();
-#elif UNITY_2020_2_OR_NEWER
-            list.ClearCacheRecursive();
+#else
+            ClearCacheMethod?.Invoke(list, Array.Empty<object>());
 #endif
         }
     }

